starlette_jsonapi/resource.py

```diff
@@ -188,6 +199,13 @@ async def to_response(self, data: dict, meta: dict = None, *args, **kwargs) -> J
             *args, **kwargs,
         )
 
+    async def paginate_request(self, object_list: Sequence) -> Pagination:
+        assert self.pagination_class is not None
```

Comment:
I think we should raise an Exception here, and reserve assertions for testing and development CLIs.